doc/man/scons.xml

```diff
@@ -1162,12 +1162,15 @@ This implies
   </varlistentry>
 
   <varlistentry>
-  <term><option>--install-sandbox=<replaceable>path</replaceable></option></term>
+  <term><option>--install-sandbox=<replaceable>INSTALL_SANDBOX</replaceable></option></term>
```

Comment:
I'd avoid using all caps here.
In most of the docs all CAPS==environment variable (unless you can also set that and have the same affect?)

Reply:
it was made to match what the helpmsg is from setting the option, in `Tool/install.py`:
```
        AddOption('--install-sandbox',
                  dest='install_sandbox',
                  type="string",
                  action="store",
                  help='A directory under which all installed files will be placed.')
```
That doesn't set it explicitly, so the default takes over - which is to all-caps the name of the destination.  In other words, the help emitted looks like this:
```
    Local Options:
      --package-type=PACKAGE_TYPE
                                  The type of package to create.
      --install-sandbox=INSTALL_SANDBOX
                                  A directory under which all installed files will
                                    be placed.
```
This is easy enough to adjust to something else, I wasn't thinking about what you commented on.